packages/ui-components/src/components/getting-started/getting-started-step-overview.ts

```diff
@@ -0,0 +1,94 @@
+import { BaseElement } from '../../base';
+import { TemplateResult, css, html } from 'lit';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+import TrailOne from '../../assets/trail-1.svg?raw';
+import TrailTwo from '../../assets/trail-2.svg?raw';
+import TrailThree from '../../assets/trail-3.svg?raw';
+
+export class GettingStartedStepOverview extends BaseElement {
+  static styles = [
+    ...BaseElement.styles,
+    css`
+      .grid-template {
+        grid-template-rows: 250px 250px 390px;
+        grid-template-columns: 1fr 1fr 200px 1fr 1fr;
+      }
+
+      .trail-one {
+        margin: 0 0 -140px 20px;
+      }
+
+      .trail-two {
+        margin: 0 20px -15px 0;
+      }
+
+      .trail-three {
+        margin: 0 0 -110px -80px;
+      }
+    `,
+  ];
+  steps: { title: string; description: TemplateResult }[] = [
+    {
+      title: '🔒 Sign in through Github',
+      description: html`
+        You should see a list of all your <u>public</u> repositories.
+      `,
+    },
+    {
+      title: '✅ Enable your repository',
+      description: html`
+        Click <u>enable</u> next to the repository you want to use with
+        <i>Stryker Dashboard</i>.<br /><br />
+        A token is shown that will be used to upload your report later. Don't
+        worry if you lose it though, you can always regenerate it.
+      `,
+    },
+    {
+      title: '📊 Upload your report',
+      description: html`
+        Use your favourite Stryker distribution to generate a steps to add your
+        token to the report. Follow <a href="/some-url">these</a> steps to add
+        your token to the report. This will make sure that <i>Dashboard</i> can
+        find it!
```

Comment:
We do not add a 'token' to the report, we use the token to authenticate with Stryker Dashboard.

Reply:
fixed in: https://github.com/JortWillemsen/stryker-dashboard-redesign/pull/20